workers/unity/Assets/Playground/Scripts/Worker/AbstractMobileClientWorker.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+using Playground;
+using UnityEngine;
+using UnityEngine.UI;
+
+public abstract class AbstractMobileClientWorker : MobileWorkerConnector
```

Comment:
Nit: Would prefer `Base` to `Abstract` as a prefix

Reply:
Why? Abstract is better imo

Reply:
`Base` should be at the end of the classname imo :P